store-cli.go

```diff
@@ -39,10 +39,26 @@ func finalRecover() {
 	successExit()
 }
 
+// Skip cache action for PR jobs (event, pipeline scope)
+func skipCache(storeType, scope, action string) bool {
+	// if is not cache, or if job is not PR
+	if (storeType != "cache" || os.Getenv("SD_PULL_REQUEST") == "") {
+		return false
+	}
```

Comment:
Do we need this block? Can just add ` if (storeType === "cache" && <IS_PULL_REQUEST>)` to the condition below

Reply:
The condition below return `true`, this one is short circuit for `false` case

Reply:
What I meant was, just have one condition `if (<IS_PULL_REQUEST> && storeType === "cache" && scope === "pipeline" && action != "get") { return true }`, but this should be ok